modules/misc/news.nix

```diff
@@ -45,6 +45,13 @@ let
       id = lib.mkDefault (builtins.hashString "sha256" config.message);
     };
   });
+
+  isNixFile = n: v: v == "regular" && lib.hasSuffix ".nix" n;
```

Comment:
Changed from `builtins.match` to `lib.hasSuffix` to avoid some of the known issues with the Nix's regex implementation and also because `nix fmt` was removing the escaping (making the regex matching more than it should).